clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -407,8 +407,6 @@ public void synchronizeKeyToPassive(PassiveSynchronizationChannel<EhcacheEntityM
 
   @Override
   public void createNew() {
-    management.init();
```

Comment:
No because the `EntityEventService` is really specific to an entity: listeners are removed:
- either when the paltform tells us an entity is destroyed
- either when a new `EntityEventService` is requested on the same consumerId, then previous ones are removed.
In the case of a failover, there were no listener registered, so the active  entity just add a new one for itself.

The boolean flag is redundant. I just did the same thing we did in our sample entity while testing. But before, we kept the `createNew()` in the active, plus this listener, so I had a "double" init. This is not the case now.